File: Lynda Courses/Programming for Non-Programmers iOS 6 with Todd Perkins/ProgrammingBasics/ProgrammingBasics/ViewController.m
//
//  ViewController.m
//  ProgrammingBasics
//
//  Created by LDC on 7/23/13.
//  Copyright (c) 2013 Todd Perkins. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    int score = 10;
    score = 11;
    score++;
    NSLog(@"score: %i", score);
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Lynda Courses/Programming for Non-Programmers iOS 9 and Swift with Todd Perkins/Challenges/Methods/Methods/ViewController.swift
//
//  ViewController.swift
//  Methods
//
//  Created by Eude Kinsley Lesperance on 4/28/16.
//  Copyright © 2016 Qlovur. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var label: UILabel!
    var clickCount: Int = 0
    
    @IBAction func didClick(sender: AnyObject) {
        clickCount += 1
        label.text = "\(clickCount)"
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

